core/src/main/scala/pureconfig/error/ConfigReaderFailures.scala

```diff
@@ -15,6 +17,37 @@ case class ConfigReaderFailures(head: ConfigReaderFailure, tail: List[ConfigRead
 
   def ++(that: ConfigReaderFailures): ConfigReaderFailures =
     new ConfigReaderFailures(head, tail ++ that.toList)
+
+  def prettyPrint(identLevel: Int = 0, tabSize: Int = 2): String = {
```

Comment:
Nitpick: I'd prefer to rename `tabSize` to `indentSize`.

Reply:
Renamed in 5f420d0